src/format.ts

```diff
@@ -115,101 +116,132 @@ function checkFormat(srcFiles: string[], baseArgs: string[], options: Options):
                           }
                         },
                         ['ignore', 'pipe', process.stderr])
-                    .stdout; 
+                    .stdout;
     out.setEncoding('utf8');
     out.on('data', (data: Buffer) => {
-      output += data; 
+      output += data;
     });
-    
-    out.on('end', () => {
-      findFormatErrorLines(output, options)
-        .then(() => {
-          resolve(output.indexOf('<replacement ') === -1 ? true : false);
-        });
+
+    out.on('end', async () => {
+      const files = output.split('<?xml version=\'1.0\'?>\n');
+      for (let i = 1; i < files.length; i++) {
```

Comment:
I would recommend using [`slice`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) so that we can just start this loop at `i = 0`.

Reply:
done